DEVDOC.md

```diff
@@ -1,5 +1,43 @@
 # GAPID Developer Documentation
 
+## Setup Golang development
+
+This project contains Golang code, but it does not have the file hierarchy of
+regular Golang projects (this is due to the use of Bazel as a build system).
+The `cmd/gofuse` utility enables to re-create the file hierarchy expected by Go
+tools:
+
+```sh
+# We assume GAPID has been checked out under ~/gapid
+cd ${HOME}/gapid
+# Make sure to build to have all compile-time generated files
+bazel build //pkg
+# Prepare a gapid-gofuse directory **outside of the gapid checkout directory**
+mkdir ${HOME}/gapid-gofuse
+# Run gofuse with the previous directory as a target, and the bazel-specific OS identifier
+OS=k8 # Pick up the right option: k8 (linux) / darwin (macos) / x64_windows (windows)
+bazel run //cmd/gofuse -- -bazelout ${OS}-fastbuild --dir ${HOME}/gapid-gofuse
+
+## In your shell init script, e.g. ~/.bashrc, set up your GOPATH:
+export GOPATH=${HOME}/gapid-gofuse
```

Comment:
This would overwrite any existing GOPATH. We should just write to append the GOPATH

Reply:
done

Reply:
The new version would end up with a leading colon if GOPATH is not yet defined. Additionally, under Windows the GOPATH is separated by semicolons.
So better just write in words "append GOPATH with ...". The user can just search how to do that on his system.

Reply:
updated